docs/en/api.md

```diff
@@ -140,13 +140,13 @@ const store = new Vuex.Store({ ...options })
 
   Replace the store's root state. Use this only for state hydration / time-travel purposes.
 
-- **`watch(getter: Function, cb: Function, options?: Object)`**
+- **`watch(fn: Function, callback: Function, options?: Object): Function`**
 
-  Reactively watch a getter function's return value, and call the callback when the value changes. The getter receives the store's state as the first argument, and getters as the second argument. Accepts an optional options object that takes the same options as Vue's `vm.$watch` method.
+  Reactively watch `fn`'s return value, and call the callback when the value changes. `fn` receives the store's state as the first argument, and getters as the second argument. Accepts an optional options object that takes the same options as Vue's `vm.$watch` method.
 
-  To stop watching, call the returned handle function.
+  To stop watching, call the returned handler function.
```

Comment:
I believe `handle` is the correct word here which means a reference to stop watch. https://en.wikipedia.org/wiki/Handle_(computing)

On the other hand, `handler` in computer science usually means a `event listener` that handles event. https://en.wikipedia.org/wiki/Handler

Indeed **handle** is quite hard to **grasp** here, maybe we can say `the returned unsubscribe function` ?

Reply:
Oh, I didn't know about the meaning of handle!
I guess `the returned unwatch (unsubscribe) function` would be clearer 🙂